--- origsrc/geany-plugins-0.18/geanylua/geanylua.c	2009-10-27 11:23:27.000000000 -0500
+++ src/geany-plugins-0.18/geanylua/geanylua.c	2010-01-28 03:44:46.075895500 -0600
@@ -42,7 +42,7 @@
 
 #include "glspi_ver.h"
 
-#define SUPPORT_LIB "libgeanylua." G_MODULE_SUFFIX
+#define SUPPORT_LIB "geanylua"
 
 #define CB_COPY(field) plugin_callbacks[i].field=glspi_geany_callbacks[i].field;
 
@@ -155,11 +155,11 @@ void plugin_init(GeanyData *data)
 	main_locale_init(LOCALEDIR, GETTEXT_PACKAGE);
 
 	geany_data=data;
-	libname=g_build_path(G_DIR_SEPARATOR_S, data->app->configdir, "plugins", "geanylua", SUPPORT_LIB, NULL);
+	libname=g_module_build_path(g_build_path(G_DIR_SEPARATOR_S, data->app->configdir, "plugins", "geanylua", NULL), SUPPORT_LIB);
 	if ( !g_file_test(libname,G_FILE_TEST_IS_REGULAR) ) {
 		gchar *libdir=get_lib_dir();
 		g_free(libname);
-		libname=g_build_path(G_DIR_SEPARATOR_S, libdir, "geany-plugins", "geanylua", SUPPORT_LIB, NULL);
+		libname=g_module_build_path(g_build_path(G_DIR_SEPARATOR_S, libdir, "geany-plugins", "geanylua", NULL), SUPPORT_LIB);
 		g_free(libdir);
 	}
 	if ( !g_file_test(libname,G_FILE_TEST_IS_REGULAR) ) {
