--- origsrc/geany-plugins-1.25/build/i18n.m4	2014-10-21 17:39:56.000000000 -0500
+++ src/geany-plugins-1.25/build/i18n.m4	2015-10-20 14:36:44.452820700 -0500
@@ -5,7 +5,7 @@ AC_DEFUN([GP_I18N],
         ALL_LINGUAS="${LINGUAS}"
     else
         if test -z "$conf_dir" ; then
-            conf_dir="."
+            conf_dir="$srcdir"
         fi
         ALL_LINGUAS=`cd "$conf_dir/po" 2>/dev/null && ls *.po 2>/dev/null | $AWK 'BEGIN { FS="."; ORS=" " } { print $[1] }'`
     fi
